keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -562,6 +562,10 @@ No.
 
 No.
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
```

Comment:
I would mention that one socket is used per KMS plugin, and that old connections are closed after new connections have been created/validated for health during an automatic reload flow.